core/internal/cluster/kafka_cluster.go

```diff
@@ -98,6 +103,19 @@ func (module *KafkaCluster) Start() error {
 	// Start main loop that has a timer for offset and topic fetches
 	module.offsetTicker = time.NewTicker(time.Duration(module.offsetRefresh) * time.Second)
 	module.metadataTicker = time.NewTicker(time.Duration(module.topicRefresh) * time.Second)
+
+	if module.groupsReaperRefresh != 0 {
+		module.groupsReaperTicker = time.NewTicker(time.Duration(module.groupsReaperRefresh) * time.Second)
+		if !module.saramaConfig.Version.IsAtLeast(sarama.V0_11_0_0) {
+			module.groupsReaperTicker.Stop()
+			module.Log.Warn("groups reaper disabled, it needs at least kafka v0.11.0.0 to get the list of consumer groups")
```

Comment:
I feel purge or cleanup would sound more intuitive.

Reply:
at least for me, `reaper` feels more intuitive, I've seen this word used widely in the Kafka and java code bases:
![image](https://user-images.githubusercontent.com/950087/93912139-2f8a5b80-fcb8-11ea-85b7-447f7bf779c7.png)